content/release-notes/2023-09-19-console.md

```diff
@@ -0,0 +1,8 @@
+### Fixes & improvements
+
+- API: Added database and role name validation to the [Create a project](https://api-docs.neon.tech/reference/createproject) endpoint to prevent creating databases and roles with protected names.
+- UI: Users with whom a Neon project is shared can now access and modify all project settings except for project deletion. Only a project owner can delete a project.
+- UI: Email addresses added when sharing a Neon project with other users are now handled in a case-insensitive manner and added to the shared project list in lowercase.
```

Comment:
These also required changes to the API.

Reply:
Thank you. Added the API prefix ("API, UI:") for both entries.